### ISSUE TITLE
How do we allow people to change information?

### ISSUE THREAD

**Issue by ryanfiller** [tidymemphis/tidymemphis#3]
I'd love take the lead on this section, I have good idea on how to use [NetlifyCMS Open Authoring](https://www.netlifycms.org/docs/open-authoring/) to tackle this. I ran the site through builtwith.com and it looks like you're already hosting on their platform, which is great. 

The downside here is that it looks like that still requires a user to have a github account, which not everyone interested in contributing to this is going to have. There's also [Git Gateway](https://www.netlifycms.org/docs/git-gateway-backend/), but it comes with its own pros and cons. 

For starters, let me setup the CMS and we can figure out authoring from there. I think we need first need to circle back to the other bullet point from the readme - 

> Create a list of what is needed right now. Information on the disease, testing sites, information on what to expect, and making sure people know what is available at any point in this process.

Can you maybe put together a rough sitemap of what types of content/collections you're expecting to see here? 
IMO a good list to start with would be 
resources - a collection of posts each containing a resource
news - a collection of posts linking out to other articles that might be important
closures - a collection of posts that each contain a restaurant/business, info about their closure and what you can do to help their employees.
map - a google map, we can look at how to mark testing centers on this as they become available
illness counter - we could maybe look at scraping some info and getting a live count of active/recovered cases? I haven't looked into this at all, but [this](https://github.com/wesbos/rona/) came across my youtube the other day.

**Comment by tidymemphis** [comment]
Yes! So happy you're hear to help. Let me know what you need me to do to let you do your thing. Lemme get back to you shortly about the site map.

**Comment by ryanfiller** [comment]
Sweet! Yeah, if you'll let me know which content types we need I can go ahead and set that up in gatsby, then if that looks good to you get it set up with Netlify so anyone can edit.

**Comment by tidymemphis** [comment]
That sounds awesome! Yeah I want this to be by the community, for the community. I finally got a good nights sleep and I'm writing up the sitemap now.

**Comment by tidymemphis** [comment]
Okay here is something a little more fleshed out

First I want to address data handling: everyone has access and full control of their own data. No data will EVER be sold. The more we work together and are open with each other, the better our chances are.

Parts of the site
1. Information on the Virus
- What is it?
- Why are we quarantining?
- How has it affected other places?
	- Link to Johns Hopkins global map: https://coronavirus.jhu.edu/map.html
	- Testing data for each state available here: https://covidtracking.com/data/
- What can we expect?
	- Links to stories from other countries
- Q/A for specific questions
	- Ex: Can I go on walks? (Yes)
	
2. Memphis Economic Impacts
- interactive map of all closings? Otherwise a list for
	- Restaurant closings
	- Store closings
	- School closings etc
- Displaced workers/tip jar
	- Is there a way for people to donate to one restaurant and have the donation be dispersed among employees?

3. COVID-19 & Memphians
	How many Memphians have it? 
		current number of cases reported by the Shelby County Health Department
		number of cases reported by TDoH
	Map of cases in each county in the surrounding area
		Very important that we include Mississippi and Arkansas counties, not just Tennessee
	Requirements to get tested.
	Where can I get tested?
		List or map of testing locations
        Where are tests verified?
        How many Memphians have been tested? Are under quarantine? 
	Where are hospitals? 
	How prepared are our hospitals?
	What do hospitals need?

**Comment by ryanfiller** [comment]
This is great! Whenever my other PR lands I can go ahead and add those pages so you/github members can edit them, then go from there hooking them up to Netlify for community editing.

**Comment by tidymemphis** [comment]
what am I forgetting to do to get the new site live

**Comment by ryanfiller** [comment]
Hmm, I'm not 100% sure about that. I do get the same error running `build` locally though. I might recommend disabling `gatsby-plugin-feed-mdx` for now until you plan on actively using it.

How is your Netlify deploy set up? There's a way to get it to automatically build and deploy when you push commits to master, that way you don't have to worry about doing anything locally.

**Comment by tidymemphis** [comment]
I think that is how my Netlify is set up--usually I just push commits to master and the changes show up, but that didn't happen this time.
